decodeWays.py

```diff
@@ -12,18 +12,21 @@ def numDecodings(self, s):
         for i in range(1, len(s)):
             flag1 = self.decodeTwoDigit(s[i - 1: i + 1])
             flag2 = self.decodeDigit(s[i])
-            if flag1 == 0 and flag2 == 0 :
+            if not flag1 and not flag2:
                 return 0
-            elif flag1 == 0:
+            elif not flag1:
```

Comment:
Actually, this should be `if` not `elif`.

Reply:
I tried, this should be elif.